Samples/Swift/DaysUntilBirthday/iOS/UserProfileView.swift

```diff
@@ -57,7 +57,10 @@ struct UserProfileView: View {
           NavigationLink(NSLocalizedString("Verify My Age", comment: "Verify Age"),
                          destination: VerificationView(verifiedAgeViewModel: verifiedAgeViewModel)
           .onAppear {
-            verifiedAgeViewModel.verifyAccountDetails()
+            verifiedAgeViewModel.verifyUserAge()
+          }
+          .onDisappear {
+            verifiedAgeViewModel.verificationState = .unverified
```

Comment:
What role does `verificationState` play now (or will play in future prs)?

Reply:
`verificationState` determines what's displayed in [`VerificationView`](https://github.com/google/GoogleSignIn-iOS/blob/aef9ff79668024f8a107c8204f205bb9c4601b72/Samples/Swift/DaysUntilBirthday/Shared/Views/VerificationView.swift#L24). For example, if the account is verified then the result object properties are listed, otherwise the view tells the user the account is unverified.

Additionally, we want to avoid falsely reflecting that the user is verified if the app is still in memory and instead ensure `verifyUserAgeOver18()` is called each time the user taps on "Verify My Age" to make the token request, so setting `verificationState` to `.unverified` will satisfy these intentions.

Reply:
Following up from an IRL conversation. @brnnmrls clarified how `verifiedAgeViewModel` propagates its state changes upon verification. Resolving.